zzre/game/systems/ui/ScrBookMenu.cs

```diff
@@ -71,6 +185,20 @@ protected override void Update(float timeElapsed, in DefaultEcs.Entity entity, r
     private void HandleElementDown(DefaultEcs.Entity entity, components.ui.ElementId id)
     {
         var bookMenuEntity = Set.GetEntities()[0];
+        ref var book = ref bookMenuEntity.Get<components.ui.ScrBookMenu>();
+
+        if (book.FairyButtons.TryGetValue(id, out var fairyRow))
+        {
+            book.Sidebar.Dispose();
+            book.Sidebar = CreateSidebar(preload, entity, fairyRow, ref book);
```

Comment:
Grouping the sidebar stuff together into an intermediate parent makes disposing very easy here, good idea 👍